### PR TITLE
Updated reference to lib.widgets.

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

File: lib/snapshot/BUILD.gn [hunk 1]
```diff
@@ -38,7 +38,7 @@ if (is_fuchsia) {
     dart_deps = [
       "//garnet/public/lib/fidl/dart",
       "//lib/flutter/packages/flutter:flutter",
-      "//lib/widgets/packages/widgets:lib.widgets",
+      "//topaz/lib/widgets/packages/widgets:lib.widgets",
```
Comment (abarth) [inline lib/snapshot/BUILD.gn]: Why is this needed here?  This code shouldn't be aware of anything else in Topaz.

**Comment by abarth** [comment]
LGTM with a TODO comment about understanding why this text is here and removing it.

File: lib/snapshot/BUILD.gn (hunk 1 above)
Comment (pylaligand) [reply lib/snapshot/BUILD.gn]: I was wondering about that too. It looks like it was introduced by @rmacnak-google in https://github.com/flutter/engine/pull/3753. Ryan, what was the motivation for adding the library?

Comment (rmacnak-google) [reply lib/snapshot/BUILD.gn]: On Fuchsia we improve the startup performance of the JIT by loading and precompiling some commonly used libraries in the core snapshot. The deps here is required because of the import in //flutter/lib/snapshot/snapshot_fuchsia.dart; alternatively the import and the deps entry can both be removed and everything will still work.

I suspect we can still get a warmed core snapshot and respect the new layering by moving the core snapshot out of the engine, but I don't understand the packaging story enough to say how.

Comment (abarth) [reply lib/snapshot/BUILD.gn]: If we're doing this for performance, we should move this library from this list.  It's not reflective of the real-world performance of the system because third party developers won't be able to have their widget library optimized in this way.

Comment (pylaligand) [reply lib/snapshot/BUILD.gn]: Agreed, and removed.
